torch/csrc/jit/operator_upgraders/upgraders.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include <atomic>
+#include <iostream>
+#include <string>
+#include <unordered_map>
+
+namespace torch {
+namespace jit {
+
+#define IS_IN_POPULATION_PHASE ([] { \
```

Comment:
neat!

Reply:
Can you explain what you are trying to do?

Reply:
So `populate_upgraders_map` inserts upgrader graph entries to an originally empty map. Since we don't want to keep adding stuff to that map everytime torch.jit.load is invoked, we do this to make sure `populate_upgraders_map` is only called once.